geode-core/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommandDUnitTest.java

```diff
@@ -121,6 +124,51 @@ public void testCreateRegionWithNoCompressor() throws Exception {
     });
   }
 
+  @Test
+  public void testCreateRegionWithSubregion() throws Exception {
+    String regionName = testName.getMethodName();
+    gfsh.executeAndAssertThat("create region --name=" + regionName + " --type=REPLICATE")
+        .statusIsSuccess();
+    String subRegionName = "subregion";
+    String subRegionPath = regionName + "/" + subRegionName;
+
+    gfsh.executeAndAssertThat("create region --name=" + subRegionPath +
+        " --type=REPLICATE")
+        .statusIsSuccess();
+
+    String subSubRegionName = "subregion2";
+    String subSubRegionPath = regionName + "/" + subRegionName + "/" + subSubRegionName;
+    gfsh.executeAndAssertThat("create region --name=" + subSubRegionPath +
+        " --type=REPLICATE")
+        .statusIsSuccess();
+
+    server1.invoke(() -> {
+      Cache cache = ClusterStartupRule.getCache();
+      Region region = cache.getRegion(regionName);
+      assertThat(region).isNotNull();
+
+      Region subRegion = cache.getRegion(subRegionPath);
+      assertThat(subRegion).isNotNull();
+
+      Region subSubRegion = cache.getRegion(subSubRegionPath);
+      assertThat(subSubRegion).isNotNull();
+    });
+
+    locator.invoke(() -> {
```

Comment:
in addition to verify the resulting xml, you can also start a third server, and verifies that on the 3rd server the region "regionName/subRegionName/subSubRegionName" exists.